### ISSUE TITLE
Headless Operation (ie: On BBB)

### ISSUE THREAD

**Issue by tkuester** [minoca/os#17]
I would love to be able to use a serial cable to access the BBB.

Since I'm an embedded developer, most of my "administrative" work is done over SSH and serial cables. It would be nice to be able to edit a text file somewhere, and assign a hardware serial port to the console!

**Comment by evangreen** [comment]
I agree, and we've had a couple of requests for this now. We do have code to support these UARTs since we currently use this port for the kernel debugger. The work needed is basically to write the interrupt-driven driver version of the UART, which can then light it up for terminal access.

This might be a good starter project for a budding community volunteer if anyone out there wants to tackle it...
